qiskit/dagcircuit/dagdependency.py

```diff
@@ -135,10 +137,23 @@ def calibrations(self, calibrations):
         """
         self._calibrations = defaultdict(dict, calibrations)
 
+    @property
+    def multi_graph(self):
+        """Return the inner :class:`~retworkx.PyDiGraph` used for the DAG representation."""
+        return self._multi_graph
+
     def to_networkx(self):
         """Returns a copy of the DAGDependency in networkx format."""
         # For backwards compatibility, return networkx structure from terra 0.12
         # where DAGNodes instances are used as indexes on the networkx graph.
+        warnings.warn(
+            "The to_networkx() method is deprecated and will be removed in a future release."
+            "Instead you can leverage the retworkx graph representation of the dag via the "
+            "multi_graph attribute",
+            DeprecationWarning,
+            stacklevel=2,
+        )
```

Comment:
_Technically_ we're not meant to issue this warning til the next Terra release, since we're only adding the alternative in this same release.  But equally, I don't think anybody's using `DAGDependency` anyway.